client/src/main/java/com/vaadin/client/widgets/Grid.java

```diff
@@ -8953,6 +9383,15 @@ public HandlerRegistration addSelectionAllowedHandler(
         return addHandler(handler, GridSelectionAllowedEvent.TYPE);
     }
 
+    /**
+     * Register a row height changed handler to this Grid. The event for this
+     * handler is fired when the row height is changed in the Grid's header,
+     * body or footer.
+     *
+     * @param handler
+     *            the handler for the event
+     * @return the registration for the event
```

Comment:
"the registration for the event" --> "the registration for the handler" ?

Reply:
Fixed all occurrences.